HTML/html_completions.py

```diff
@@ -365,4 +367,3 @@ def test_expand_tag_attributes(self):
         # should give just one completion, and suppress word based completion
         self.assertEqual(completion_list, [('td.class', '<td class="class">$1</td>$0')])
         self.assertEqual(flags, sublime.INHIBIT_WORD_COMPLETIONS | sublime.INHIBIT_EXPLICIT_COMPLETIONS)
-
```

Comment:
Looks good - might want to keep this in - yo can set `"ensure_newline_at_eof_on_save": true,` in your prefs to prevent this from happening